src/mod/endpoints/mod_sofia/sofia_media.c

```diff
@@ -162,11 +163,11 @@ char *sofia_media_get_multipart(switch_core_session_t *session, const char *pref
 					int i = 0;
 
 					for(i = 0; i < hi->idx; i++) {
-						process_mp(session, &stream, boundary, hi->array[i], isup, isup_len, !!sdp);
+						process_mp(session, &stream, boundary, hi->array[i], isup, isup_len, !!sdp, allow_sdp);
```

Comment:
We can reuse the function parameter drop_sdp having this variables before calling this method
```
switch_bool_t allow_custom_sdp = switch_true(switch_channel_get_variable(channel, "allow_custom_sdp"));
switch_bool_t drop_sdp = !!sdp && !allow_custom_sdp;
...
process_mp(session, &stream, boundary, value, isup, isup_len, drop_sdp);
```

Reply:
Done